### PR TITLE
MGMT-915 - adjust hosts section of installconfig to reflect the corre…

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…ct number of hosts intances

### CODE REVIEW

File: internal/installcfg/installcfg.go [hunk 1]
```diff
@@ -134,185 +143,133 @@ func getMachineCIDR(cluster *models.Cluster) (string, error) {
...
+	}
+	hwMac[len(hwMac)-1] = hwMac[len(hwMac)-1] + byte(count)
+	return hwMac.String(), nil
+}
+
+func setPlatformInstallconfig(cluster *models.Cluster, cfg *InstallerConfigBaremetal) error {
+	// set hosts
+	numMasters := countHostsByRole(cluster, "master")
+	numWorkers := countHostsByRole(cluster, "worker")
+	logrus.Infof("Number of masters in cluster i %d, number of worker is %d", numMasters, numWorkers)
```
Comment (romfreiman) [inline internal/installcfg/installcfg.go:213]: workers

File: internal/installcfg/installcfg.go [hunk 2]
```diff
@@ -134,185 +143,133 @@ func getMachineCIDR(cluster *models.Cluster) (string, error) {
...
+	masterCount := 0
+	workerCount := 0
+	hosts := make([]host, numWorkers+numMasters)
+
+	// dummy MAC and port, once we start using real BMH, those values should be set from cluster
+	dummyMAC := "00:aa:39:b3:51:10"
```
Comment (romfreiman) [inline internal/installcfg/installcfg.go:219]: maybe dont set same mac but ++ every time?

Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go:219]: that's what i do, see function getDummyMAC

File: internal/installcfg/installcfg.go (hunk 1 above)
Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go:213]: removed this printout , it was for debug purposes

Comment (romfreiman) [reply internal/installcfg/installcfg.go]: why? Lets have those logs

Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go]: Believe me , this one was really not needed :)

File: internal/installcfg/installcfg.go [hunk 3]
```diff
@@ -134,185 +143,132 @@ func getMachineCIDR(cluster *models.Cluster) (string, error) {
...
+	dummyPort := 6230
+
+	for i := range hosts {
+		logrus.Infof("Setting master, host %d, master count %d", i, masterCount)
```
Comment (eranco74) [inline internal/installcfg/installcfg.go:222]:
```suggestion
		logrus.Infof("Setting host info, host %d, master count %d", i, masterCount)
```
You are setting both workers and masters

Comment (yevgeny-shnaidman) [reply internal/installcfg/installcfg.go:222]: as explained above, this is a debug log, not needed anymore, i removed it

**Pull request merged**